bootstrap.bash

```diff
@@ -1,13 +1,15 @@
 #!/bin/bash
 
-# This script serves two purposes.  First, it can bootstrap the standalone
-# Blueprint to generate the minibp binary.  To do this simply run the script
-# with no arguments from the desired build directory.
+# This script serves two purposes.  First, it can generate bootstrapping files
+# that can be used to build the minibp binary.  To generate these files, simply
+# run this script with no arguments. The generated files will be placed in the
+# working directory (unless BUILDDIR is set to something else).
 #
-# It can also be invoked from another script to bootstrap a custom Blueprint-
-# based build system.  To do this, the invoking script must first set some or
-# all of the following environment variables, which are documented below where
-# their default values are set:
+# This script can also generate a custom bootstrap manifest (most commonly
+# called "build.ninja.in") for use in another Blueprint-based build system.
+# This is activated by the '-r' command-line option.
```

Comment:
The -r option would actually be the third purpose (and we'd use it to generate the build.ninja.in in this directory too, not just for other Blueprint-based systems.